packages/dx-react-scheduler-material-ui/src/templates/date-navigator/root.test.jsx

```diff
@@ -26,8 +26,9 @@ describe('DateNavigator', () => {
         .toMatchObject({ a: 1 });
     });
     it('should render open button', () => {
+      const mount = createMount();
```

Comment:
You should call mount as it describes from the [tutorial](https://material-ui.com/guides/testing/#createmount-options-mount). Please, add mount.cleanUp() function.

```
after(() => {
  mount.cleanUp();
});
```

BTW you should create `mount` for each test as it does for `shallow`, see the example from Material-ui site.